nixd/docs/editor-setup.md

```diff
@@ -1,4 +1,4 @@
-## We ❤️ Contributions!
+## We ❤ Contributions!
```

Comment:
why this "heart" should be changed?

Reply:
The heart is changed because there is an unrecognized character on Windows.

I've tried to revert it, but nothing has changed. 🤔 (For some reason, it remains the blue heart instead of the original red heart.)